ui/apps/platform/src/hooks/useCentralCapabilities.ts

```diff
@@ -0,0 +1,45 @@
+import { useSelector } from 'react-redux';
+import { createStructuredSelector } from 'reselect';
+
+import { selectors } from 'reducers';
+import { CentralCapabilitiesFlags } from 'types/centralCapabilities';
+import { CentralServicesCapabilities } from 'services/MetadataService';
+import React from 'react';
+
+const centralCapabilitiesSelector = createStructuredSelector<{
+    centralCapabilities: CentralServicesCapabilities;
+    isLoadingCentralCapabilities: boolean;
+    centralCapabilitiesError: null | Error;
+}>({
+    centralCapabilities: selectors.getCentralCapabilities,
+    isLoadingCentralCapabilities: selectors.getIsLoadingCentralCapabilities,
+    centralCapabilitiesError: selectors.getCentralCapabilitiesError,
+});
+
+type UseCentralCapabilityResult = {
+    isCentralCapabilityAvailable: (CentralCapabilityFlag: CentralCapabilitiesFlags) => boolean;
+    isLoadingCentralCapabilities: boolean;
+};
+
+function useCentralCapabilities(): UseCentralCapabilityResult {
+    const { centralCapabilities, isLoadingCentralCapabilities } = useSelector(
+        centralCapabilitiesSelector
+    );
+
+    const isCentralCapabilityAvailable = React.useMemo(
+        () =>
+            (CentralCapabilityFlag: CentralCapabilitiesFlags): boolean => {
```

Comment:
camelCase is more common for function argument: `centralCapabilityFlag`

Reply:
whoops, accidental copy paste action